Mixpanel/Persistence.swift

```diff
@@ -118,8 +118,21 @@ class Persistence {
 
         if !NSKeyedArchiver.archiveRootObject(object, toFile: path) {
             Logger.error(message: "failed to archive \(type.rawValue)")
+            return
         }
 
+        addSkipBackupAttributeToItem(at: path)
+    }
+
+    class private func addSkipBackupAttributeToItem(at path: String) {
+        var url = URL.init(fileURLWithPath: path)
+        var resourceValues = URLResourceValues()
+        resourceValues.isExcludedFromBackup = true
```

Comment:
sorry, forgot to add context: https://github.com/mixpanel/mixpanel-iphone/issues/565